.vscode/settings.json

```diff
@@ -4,5 +4,7 @@
   },
   "editor.quickSuggestions": {
     "strings": "on"
-  }
+  },
+  "prettier.configPath": "./prettier.config.cjs",
+  "editor.wordWrap": "on"
```

Comment:
Since Tailwind CSS is so long I think it makes sense to just have wordWrap on all the time.